modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -54,23 +57,25 @@ const outcome = (grid: Structs.RowCells[], cursor: Optional<SugarElement>): Tabl
   cursor
 });
 
+const findEditableCursorPosition = (rows: Structs.RowCells[]) =>
+  Arr.findMap(rows, (row) =>
+    Arr.findMap(row.cells, (cell) => {
+      const elem = cell.element;
+      return Optionals.someIf(isEditable(elem), elem);
+    })
+  );
+
 const elementFromGrid = (grid: Structs.RowCells[], row: number, column: number) => {
```

Comment:
Is this function name clear enough now, regarding what this should be used for?

Reply:
No one else has commented regarding this and it's still not any different to what it was previously so marking as resolved.